src/main/index.ts

```diff
@@ -126,24 +129,64 @@ Menu.setApplicationMenu(kangarooMenu(KANGAROO_FILESYSTEM));
 app.whenReady().then(async () => {
   SPLASH_SCREEN_WINDOW = createSplashWindow();
   ipcMain.handle("sign-zome-call", handleSignZomeCall);
-  ipcMain.handle('exit', () => {
+  ipcMain.handle("exit", () => {
     app.exit(0);
   });
+
+  // Check for updates
+  if (app.isPackaged && KANGAROO_CONFIG.autoUpdates) {
+    autoUpdater.allowPrerelease = true;
+    autoUpdater.autoDownload = false;
+
+    let updateCheckResult: UpdateCheckResult | null | undefined;
```

Comment:
I think one of `null` or `undefined` should be enough here

```suggestion
    let updateCheckResult: UpdateCheckResult | undefined;
```

Reply:
Actually no, `checkForUpdates()` can return null and if it fails altogether then on line 149 typescript complains as it may be undefined. So it can be all three of them and typescript wants me to be explicit there.